### PR TITLE
BACKLOG-34725 cdpdc71 shim

### PR DESCRIPTION
**Pull request by kodehash**
New Big Data Shim for CDP-DC 7.1.3

### CODE REVIEW

File: shims/cdpdc71/driver/pom.xml [hunk 1]
```diff
@@ -0,0 +1,332 @@
...
+  <packaging>bundle</packaging>
+
+  <properties>
+    <commons-lang.version>2.6</commons-lang.version>
+    <pentaho-osgi-bundles.version>9.2.0.0-SNAPSHOT</pentaho-osgi-bundles.version>
+    <org.apache.avro.version>1.8.0</org.apache.avro.version>
+    <org.apache.orc.version>1.2.3</org.apache.orc.version>
+    <org.apache.hadoop.version>3.1.1.7.1.3.0-100</org.apache.hadoop.version>
+    <org.apache.hbase.version>2.2.3.7.1.3.0-100</org.apache.hbase.version>
+    <org.apache.hive.version>2.1.1-cdh6.1.0</org.apache.hive.version>
```
Comment (kodehash) [inline shims/cdpdc71/driver/pom.xml:22]: This is the version that works for hive, with the appropriate version - 3.1.3000.7.1.3.0-100 I have been facing bundling issues

File: shims/cdpdc71/driver/pom.xml [hunk 2]
```diff
@@ -0,0 +1,332 @@
...
+    <org.apache.oozie.version>5.1.0.7.1.3.0-100</org.apache.oozie.version>
+    <pig.version>0.17.0-cdh6.1.0</pig.version>
```
Comment (kodehash) [inline shims/cdpdc71/driver/pom.xml:24]: There are no pig components in the CDPDC7.1 cluster, should we keep it as it is ?

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 15m 26s
Build command:
```console
mvn clean verify -B -e -Daudit -amd -pl \
shims,shims/cdpdc71,shims/cdpdc71/driver,shims/cdpdc71/kar,shims/cdpdc71/pmr,shims/mapr61,shims/mapr61/driver,shims/mapr61/kar,shims/mapr61/pmr
```
---
:ok_hand: All tests passed!
 
Tests run: **30**, Failures: **0**, Skipped: **0** &nbsp;&nbsp; [Test Results](http://wingman.pentaho.net/job/projects/job/big-data-hadoop-shims/job/PR-1182/1/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 13m 55s
Build command:
```console
mvn clean verify -B -e -Daudit -amd -pl \
shims,shims/cdpdc71,shims/cdpdc71/driver,shims/cdpdc71/kar,shims/cdpdc71/pmr,shims/mapr61,shims/mapr61/driver,shims/mapr61/kar,shims/mapr61/pmr
```
---
:ok_hand: All tests passed!
 
Tests run: **30**, Failures: **0**, Skipped: **0** &nbsp;&nbsp; [Test Results](http://wingman.pentaho.net/job/projects/job/big-data-hadoop-shims/job/PR-1182/2/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

**Pull request merged**